handlers/load_test_handler.go

```diff
@@ -141,12 +148,22 @@ func (h *Handler) jsonToMap(headersString string) *map[string]string {
 
 // LoadTestHandler runs the load test with the given parameters
 func (h *Handler) LoadTestHandler(w http.ResponseWriter, req *http.Request, prefObj *models.Preference, user *models.User, provider models.Provider) {
-	// if req.Method != http.MethodPost && req.Method != http.MethodGet {
-	// 	w.WriteHeader(http.StatusNotFound)
-	// 	return
-	// }
+	body, err := ioutil.ReadAll(req.Body)
+	if err != nil {
+		msg := "unable to read request body"
+		err = errors.Wrapf(err, msg)
+		logrus.Error(err)
+		http.Error(w, msg, http.StatusInternalServerError)
```

Comment:
Use the new [meshkit error](https://docs.meshery.io/project/contributing-error)?

Reply:
Point me to an example.

Reply:
hey @piyushsingariya this [PR ](https://github.com/meshery/meshery/pull/3373)might be of your help just check the mesh_ops_handler.go